WordPress/gutenberg
# adds default alt values to inline images and the image block

Sets default alt values to fix #1520 
Previous conversation can be found here: https://github.com/WordPress/gutenberg/pull/10482

How to test:
create image block
add image with no alt value
inspect image to see that alt value is now set as a readable string "This image has an empty alt attribute, its file name is "FILENAME""
